services/ui-src/src/components/tables/EntityRow.tsx

```diff
@@ -38,15 +45,27 @@ export const EntityRow = ({
   }, [report]);
 
   const entityFields = () => {
-    const fields: string[] =
-      reportType === ReportType.MLR
-        ? [
-            report_planName,
-            report_programName,
-            eligibilityGroup(entity),
-            reportingPeriod,
-          ]
-        : [name];
+    let fields: string[] = [];
+    switch (reportType) {
```

Comment:
change the fields to display in the table by report type (at the moment this is only used in MCPAR for quality measures, in MLR for programs, and in NAAAR for plan compliance)